runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -691,7 +691,7 @@ else if (event.getMessage().contains(KILLED_TELEBLOCK_OPPONENT_TEXT))
 				int wave = Integer.parseInt(matcher.group(1));
 				if (wave == 1)
 				{
-					config.tzhaarStartTime(now);
+					config.tzhaarStartTime(now.minus(Duration.ofSeconds(6)));
```

Comment:
This would affect timers for both fight caves and inferno? If so, this isn't correct.